go.mod

```diff
@@ -107,3 +107,5 @@ require (
 )
 
 go 1.18
+
+replace github.com/codeready-toolchain/toolchain-common => github.com/mfrancisc/toolchain-common v0.0.0-20230105093252-b18c1ea3a028
```

Comment:
reminder ;)

Reply:
changed in [21db486](https://github.com/codeready-toolchain/host-operator/pull/722/commits/21db486561798e906aed27c873c750db9e1b0bcf)